### PR TITLE
feat/card-item

### PR DESCRIPTION
**Pull request by airamel**
As an user of cloth catalog I want to see the items in a styled way where I can find more appealing to buy an item.

### CODE REVIEW

File: src/app/Domain/state/cloth/cloth.reducer.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ const _clothReducer = createReducer(
     (state: ClothState, { spreadsheet }) => {
       // Map the values from the spreadsheet to an array of Cloth items, removes the first item which contains the
       // headers of the spreadsheet
+      const value = 'Disponible';
```
Comment (marcmelchor) [inline src/app/Domain/state/cloth/cloth.reducer.ts]: You are creating this constant, but you are not using it.

File: src/app/Domain/state/cloth/cloth.reducer.ts [hunk 2]
```diff
@@ -29,9 +30,10 @@ const _clothReducer = createReducer(
             size: currentValue[9],
             price: currentValue[11],
             status: currentValue[13],
+            photo: currentValue[15],
           });
-          return accumulator;
-        }, []).slice(1);
+          return accumulator.filter(accumulator => accumulator.status == "Disponible");
```
Comment (marcmelchor) [inline src/app/Domain/state/cloth/cloth.reducer.ts]:
```suggestion
          return accumulator.filter(accumulator => accumulator.status == this.value);
```
You can use here the variable. Nevertheless, the name `value` doesn't really makes any sense to what it actually does.

File: src/app/UI/widgets/card/card.component.html [hunk 3]
```diff
@@ -1 +1,32 @@
-<p>card works!</p>
+<!--span>{{ data.cloth }} {{ data.brand }} {{ data.description }} {{ data.color }} {{ data.size }} {{ data.price }} {{ data.status }} {{ data.photo}}</span-->
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.html]: Why are you keeping a commented line, you should actually deleted it.

File: src/app/UI/widgets/card/card.component.html [hunk 4]
```diff
@@ -1 +1,32 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Card Clothes</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&display=swap">
+  <link rel="stylesheet" href="card.component.scss">
+</head>
+<body>
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.html]:
```suggestion
<div class="container">...</div>
```
This is totally wrong, if you inspect your app, you can see that you have now two `html` tags, two `head` and two `body`. Please get rid off them.

File: src/app/UI/widgets/card/card.component.html [hunk 5]
```diff
@@ -1 +1,32 @@
...
+          <li class="chip">{{data.brand}}</li>
+          <li class="chip">{{data.size}}</li>
+          <li class="chip">{{data.price}}</li>
+        </ul>
+      </div>
+    </article>
+  </div>
+  
+</body>
+</html>
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.html]: Removed them too, due to the message above.

File: src/app/UI/widgets/card/card.component.scss [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+
+body {
+  background-color: $background;
+}
+
+* {
+  margin: 0px;
+  padding: 0px;
+}
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.scss]:
```suggestion
body {
  background-color: $background;
}

* {
  margin: 0;
  padding: 0;
}
```

This section should be added on `src/styles.scss` also, when you have `0px` attributes, the notation should be just `0`.

**Review by marcmelchor — CHANGES_REQUESTED** [review]
Hello @airamel, I have made my code review, please let me know if you have any doubts or questions.

File: src/app/UI/widgets/card/card.component.scss [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  &:hover {
+    box-shadow: rgba(0, 0, 0, 0.3)
+    0px 5px 15px;
+  }
+}
+
+.background {
+  flex: 1 1 250px;
+
+  img {
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.scss]: You're not following the `BEM` principle.

File: src/app/Domain/state/cloth/cloth.reducer.ts (hunk 2 above)
Comment (airamel) [reply src/app/Domain/state/cloth/cloth.reducer.ts]: In the line 34 the slice index would be 0 if not is excluding the first element.

File: src/app/UI/widgets/card/card.component.scss [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+  p {
+    font-size: clamp(1rem, 1.7vw, 1.1rem);
+    font-weight: 400;
+    margin: 6px 0;
+  }
+
+  a {
+    color: $primary-violet;
+  }
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.scss]: You're not following the `BEM` principle.

File: src/app/UI/widgets/card/card.component.scss [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+    color: $gray;
+  }
+}
+
+
+
+
+
+
+
```
Comment (marcmelchor) [inline src/app/UI/widgets/card/card.component.scss]: Remove unneeded lines.

**Review by marcmelchor — CHANGES_REQUESTED** [review]
@airamel, please resolve the comments as soon as possible.

**Review by marcmelchor — APPROVED** [review]

**Pull request merged**